README.md

```diff
@@ -1,82 +1,53 @@
-Terraform Provider for AWS
-==================
+<a href="https://terraform.io">
+    <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" alt="Terraform logo" title="Terrafpr," align="right" height="50" />
+</a>
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+# Terraform Provider for AWS 
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+[![Build Status][travis-badge]][travis]
+[![Forums][discuss-badge]][discuss]
```

Comment:
I wanted to highlight the 'official' channel for comms, not sure if a badge is the right place.